### DISCUSSION TITLE
Missing details for failed cases

### DISCUSSION THREAD

**Discussion by TheDMon** [sorry-cypress/sorry-cypress#585] [Q&A]
I don't see such additional context being recorded/presented in the Sorry Cypress dashboard in comparison to the html report generated through cypress-machawesome-reporter plugin. This is definitely helpful to know why the test case has failed. Is there a way to enable such details in the dashboard?

![image](https://user-images.githubusercontent.com/33923273/173557335-6b9bfc1e-d7bc-47d7-a4a3-647bb62f9a4d.png)

**Comment by agoldis** [comment]
yeah, it is possible - the data is there, it just needs UI / backend work
